Lib/cgitb.py

```diff
@@ -124,7 +124,7 @@ def html(einfo, context=5):
         args, varargs, varkw, locals = inspect.getargvalues(frame)
         call = ''
         if func != '?':
-            call = 'in ' + strong(func) + \
+            call = 'in ' + strong(pydoc.html.escape(func)) + \
```

Comment:
You can use `pydoc.html.escape()` also in line 285.

Reply:
Line 285 is not modified by the patch but I agree it's good to reuse ```pydoc.html.escape()``` there.
I committed the requested change.